deploy/template/nginx_site.conf

```diff
@@ -16,6 +16,17 @@ server {
         }
     }
 
+    location /auth {
```

Comment:
I'm not sure about this. Because this is not an API really -- we don't get or post any data.
But I can change it to `/api/0/auth`, maybe this will be more appropriate.